### PR TITLE
Add 'make check' to verify a few common setup requirements

### PR DESCRIPTION
**Pull request by soulshake**
After running into a couple of hangups getting to `make plan`, I thought it might be helpful to add a `make check` target to verify some required envvars and credentials.

Thoughts?

### CODE REVIEW

File: bin/check-credentials [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  stderr_echo "$@" && exit 1
+}
+
+stderr_echo() {
+  >&2 echo "$@"
+}
+
+check_heroku() {
+    endpoints="
+        travis-pro-web-staging
```
Comment (igorwwwwwwwwwwwwwwwwwwww) [inline bin/check-credentials]: maybe change this to `travis-pro-staging`? that way the arbitrary apps we check follow some sort of pattern.

**Comment by bogdanap** [comment]
Isn't it worth merging this straight into master rather than meat-worker-rollout?

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
As per slack, since `bin/travis-worker-verify-config` is run as a local-exec via terraform, merging them may not be so straight forward. So nevermind that, I think we can move forward with this as-is!

@bogdanap I'd generally agree that targeting master and then merging master back into branches is preferable. 👍 should be better once `meat-worker-rollout` lands in master.

**Comment by soulshake** [comment]
Updated to merge directly to master.

**Pull request merged**